source/edge-server/connect.txt

```diff
@@ -170,58 +207,155 @@ modifications you can make to the :github:`Swift template app
    .. step:: Run the App
 
       Now you can build and run the app. When you log in, the app connects 
-      to the Edge Server. When you create new Items, those sync with the 
-      Edge Server. The Edge Server then syncs with the Atlas App Services App.
+      to the Edge Server instance. When you create new Items, those sync with 
+      the instance. The instance then syncs with the Atlas App Services App.
 
-      You can check the status of the Edge Server from your host.
+      You can check the status of the Edge Server instance from your host.
       With the client running, you can see ``"num_local_clients": 1`` in 
-      the Edge Server status. You can also see error messages reflected
+      the instance status. You can also see error messages reflected
       in the sync status.
 
-      After the next time the Edge Server syncs with your App Services App,
-      you can see synced changes reflected in the linked Atlas collection.
+      After the next time the Edge Server instance syncs with your App Services 
+      App, you can see synced changes reflected in the linked Atlas collection.
 
       .. important::
 
          If you have previously connected directly from the client to your 
          App Services App with Device Sync, you may need to clear state on the 
          simulator or device. Resetting your cache enables your client to 
-         connect to the Edge Server.
+         connect to the Edge Server instance.
 
          In a console app, delete the ``mongodb-realm`` directory. If you're
          using an iOS or Android simulator or device, uninstall and reinstall
          the app.
 
 .. _edge-server-wireprotocol-connect:
 
-Connect to the Edge Server with a MongoDB URI
----------------------------------------------
+Connect to the Edge Server with MongoDB Drivers and Tools
+---------------------------------------------------------
 
 You can connect to the Edge Server using standard MongoDB Drivers and tools.
 Clients use a specialized MongoDB URI connection string to connect to 
 Edge Server and send requests.
 
-The wire protocol server is on by default. You can connect to it via the 
-host machine using this MongoDB URI:
+The Wire Protocol server accepts these incoming connections, and is on by 
+default. You can connect to the host machine with a MongoDB URI.
```

Comment:
I don't think we actually support disabling the Wire Protocol server currently, @sudssm is this something we want? I think we discussed this before.

Reply:
I think this was language we added a while ago, as it seemed worth noting since we did not require authentication to connect via the Wire Protocol. But since that is no longer the case, I'll just remove the "on by default" language from here. If you decide later there's something we want to call out around this, I'm happy to update it.